src/themes/components/modal.ts

```diff
@@ -0,0 +1,102 @@
+import { mode } from '@chakra-ui/theme-tools';
+import {
+  createMultiStyleConfigHelpers,
+  defineStyle,
+} from '@chakra-ui/styled-system';
+import { modalAnatomy as parts } from '@chakra-ui/anatomy';
+
+import { opacity } from '../shared/opacity';
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(parts.keys);
+
+const baseStyleOverlay = defineStyle({
+  bg: 'black',
+  zIndex: 'modal',
+  opacity: `${opacity[40]} !important`,
+});
+
+const baseStyleDialogContainer = defineStyle({
+  display: 'flex',
+  zIndex: 'modal',
+  justifyContent: 'center',
+  alignItems: 'center',
+  overflow: 'hidden',
+  overscrollBehaviorY: 'none',
+});
+
+const baseStyleDialog = defineStyle({
+  borderRadius: 'sm',
+  bg: mode('white', 'gray.700'),
+  color: 'inherit',
+  my: '16',
+  zIndex: 'modal',
+  boxShadow: mode('lg', 'dark-lg'),
+});
+
+const baseStyleHeader = defineStyle({
+  px: 'lg',
+  py: 'xl',
+  fontSize: 'md',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  fontWeight: 'semibold',
+  position: 'relative',
+});
+
+const baseStyleCloseButton = defineStyle({
+  width: '18px',
```

Comment:
![Bildschirmfoto 2022-12-20 um 15 32 37](https://user-images.githubusercontent.com/71456764/208691093-8c421c88-65a1-4bd2-94a6-581bdec3d313.png)
in the design it's 24px. All our icons usually use 24px